lms/djangoapps/courseware/tests/helpers.py

```diff
@@ -33,7 +33,7 @@
 from xmodule.modulestore.django import modulestore  # lint-amnesty, pylint: disable=wrong-import-order
 from xmodule.modulestore.tests.django_utils import TEST_DATA_MONGO_MODULESTORE, ModuleStoreTestCase  # lint-amnesty, pylint: disable=wrong-import-order
 from xmodule.modulestore.tests.factories import CourseFactory, BlockFactory  # lint-amnesty, pylint: disable=wrong-import-order
-from xmodule.tests import get_test_descriptor_system, get_test_system  # lint-amnesty, pylint: disable=wrong-import-order
+from xmodule.tests import get_test_block_system, get_test_system  # lint-amnesty, pylint: disable=wrong-import-order
```

Comment:
We didn't rename this one in `xmodule/tests/__init__.py` (and let's keep it as `get_test_descriptor_system`).

Reply:
reverted